terraform/variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "model_name" {
+  description = "Name of Juju model to deploy application to"
+  type        = string
+  default     = ""
+}
+
+variable "channel" {
+  description = "The channel to use when deploying a charm "
+  type        = string
+  default     = "latest/stable"
+}
+
+variable "grafana-config" {
```

Comment:
Please see the proposal regarding config options above.

Reply:
I commented above. Besides, as I remember from one of the dailies, expanding config-options was not our choice.

Reply:
Yes, let's close this.